DeveloperGame/DeveloperGame/ComputerGeekGame.cs

```diff
@@ -25,8 +45,37 @@ public void Run()
             {
                 PrintLeaderBoard();
             }
-           
-        } while (!UserChoseToQuit(userMenuChoice));
+            if (UserChoseToQuit(userMenuChoice))
+            {
+                return;
+            }
+        }
+    }
+
+    private void ReadScoresFromFile()
+    {
+        StreamReader reader = File.OpenText(_scoreBoardfile);
+
+        string line = reader.ReadLine();
+
+        while (line != null)
+        {
+            string[] scoreAndName = line.Split(' ');
+
+            if (scoreAndName.Length == 2)
+            {
+                bool isItConvertible = int.TryParse(scoreAndName[0], out int result);
```

Comment:
Also you don't have to do this, but I would recommend putting the name first, then score, then time. I think this is the convention for games

Reply:
Ok, sounds good.

Reply:
Fixed.